tutorials/basic-topics/mod.md

```diff
@@ -1,10 +1,83 @@
 ---
 title: 'MOD (1e9 + 7)'
-description: ''
+description: 'Mod 1e9+7'
 hide_table_of_contents: true
 draft: true
 keywords:
   - leetcode
   - tutorial
   - MOD
----
\ No newline at end of file
+---
+
+<TutorialAuthors names="@tannudaral"/>
+
+## Overview
+
+Sometimes, the answer to a problem is a very large number but it is enough to output it "modulo m", i.e., the remainder when the answer is divided by m (for example, "modulo 1e9 + 7"). The idea is that even if the actual answer is very large, it suffices to use the data types int and long long. Many languages do not support large-integer arithmetic, so this method avoids integer overflow.
```

Comment:
I think this intro is from CP handbook. If so, don't copy the exact content from other sources, use your own words to describe it.